.env.local

```diff
@@ -1 +1,5 @@
-REACT_APP_API_URL=http://localhost
\ No newline at end of file
+REACT_APP_API_URL=http://localhost
```

Comment:
of course this must change before merging :-)

Reply:
env.local file is for "local development"... where is the problem? :)